recipes/gperf/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+
+class GperfConan(ConanFile):
+    name = "gperf"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gperf"
+    description = "GNU gperf is a perfect hash function generator"
+    topics = ("conan", "gperf", "hash-generator", "hash")
+    settings = "os_build", "arch_build", "compiler"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os_build == "Windows" and tools.os_info.is_windows and self.settings.compiler == "gcc"
```

Comment:
I still think the `tools.os_info.is_windows` should be dropped because the test for Windows is already done against os_build. And this does not support mingw on Linux.

Reply:
This is later used to determinate if `win_bash` is set to `true`. It is not supposed to support mingw on Linux, hence the name `_is_mingw_windows`